renovate.json

```diff
@@ -140,6 +140,12 @@
         "postgres"
       ],
       "allowedVersions": "15-alpine"
+    },
+    {
+      "matchDepNames": [
+        ".*cypress.*"
```

Comment:
are we sure this only targets the Cypress image tag but not npm dependencies like `eslint-plugin-cypress`?

Reply:
Jup, done.
Result can be seen here: https://github.com/BacLuc/ecamp3/commit/7f5b210092fbb965496142de2ffa81b547b2e78a

The auto update of the cypress docker image does not work yet.
-> planned to do in another PR
(for now we rather don't want cypress to be updated)